plugins/source/launchdarkly/CHANGELOG.md

```diff
@@ -0,0 +1 @@
+# Changelog
```

Comment:
Our bot appends to this file (and creates it if doesn't exist), so we should remove this

Reply:
Done